tests/storage/guestfs.go

```diff
@@ -200,16 +225,16 @@ var _ = SIGDescribe("[rfe_id:6364][[Serial]Guestfs", func() {
 			Expect(err).ToNot(HaveOccurred())
 			setGroup = false
 		})
-
 		AfterEach(func() {
-			err := virtClient.CoreV1().PersistentVolumeClaims(util.NamespaceTestDefault).Delete(context.Background(), pvcClaim, metav1.DeleteOptions{})
-			Expect(err).ToNot(HaveOccurred())
+			podName = libguestsTools + pvcClaim
```

Comment:
podName is set twice both by the test and the aftereach, this might lead to future confusion